app/ide-desktop/lib/dashboard/src/authentication/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "enso-studio-authentication",
+  "version": "1.0.0",
+  "type": "module",
+  "main": "./src/index.tsx",
+  "exports": "./src/index.tsx",
+  "devDependencies": {
+    "tailwindcss": "^3.2.6",
+    "typescript": "^4.9.3"
+  },
+  "dependencies": {
+    "@aws-amplify/auth": "^5.1.8",
+    "@aws-amplify/core": "^5.0.14",
+    "@fortawesome/free-brands-svg-icons": "^6.3.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
+    "opener": "^1.5.2",
```

Comment:
why opener is istalled here? perviously this lib was in lib/client/package.json

Reply:
and the function itself is being used in lib/client/src/authentication.ts

Reply:
Fixed, accidentally added to the wrong package.